src/pharmpy/modeling/odes.py

```diff
@@ -206,7 +208,9 @@ def set_first_order_elimination(model: Model):
     return model
 
 
-def add_bioavailability(model: Model, add_parameter: bool = True):
+def add_bioavailability(
```

Comment:
Do you need the option here to set the initial estimate since we have functions for that? Was thinking so that it's equivalent to `add_lag_time()`, and also the initial estimate is only relevant if you add a parameter

Reply:
In order to follow the same logic as for `add_lag_time`, that option has been removed in order to stay consistent.